### PR TITLE
Create model migration controller database users

### PR DESCRIPTION
**Pull request by sebasam**
### Summary

- Model user added by sequelize-cli.
- Migration user added by sequelize-cli.
- Controllers folder created and userControllers.js file added.

### CODE REVIEW

**Comment by maxxCeballos** [comment]
No usar la notacion camelCase para nombrar el archivo controlador, utilizar user.controller.js o user.js dentro de la carpeta controller ya define que es un controller.

File: migrations/20200921211634-create-user.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ module.exports = {
       },
       image: {
         type: Sequelize.STRING
+
```
Comment (maxxCeballos) [inline migrations/20200921211634-create-user.js]: remover este espacio que se agrega

File: migrations/20220715181113-create-user.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+module.exports = {
```
Comment (maxxCeballos) [inline migrations/20220715181113-create-user.js]: para que se hace de nuevo otro migration?

File: migrations/20200921211634-create-user.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ module.exports = {
   up: async (queryInterface, Sequelize) => {
     await queryInterface.createTable('Users', {
       id: {
-        allowNull: false,
+        notNull: true,
```
Comment (maxxCeballos) [inline migrations/20200921211634-create-user.js]: existe la propiedad notNull?

**Review by maxxCeballos — CHANGES_REQUESTED** [review]

**Comment by sebasam** [comment]
### Summary

1.  Controller filename has been fixed.
2.  Duplicate migration deleted and notNull chaned to allowNull.

File: migrations/20220715181113-create-user.js [hunk 4]
```diff
@@ -9,40 +9,44 @@ module.exports = {
         type: Sequelize.INTEGER
       },
       firstName: {
-        type: Sequelize.STRING
+        type: Sequelize.STRING,
+        allowNull: false
       },
-      lastName: {
-        type: Sequelize.STRING
+      LastName: {
```
Comment (maxxCeballos) [inline migrations/20220715181113-create-user.js]: usar minusculas como primera letra de atributo

File: models/user.js [hunk 5]
```diff
@@ -10,20 +10,22 @@ module.exports = (sequelize, DataTypes) => {
...
     static associate(models) {
-      User.belongsTo(models.Role, {as: 'role'});
+      // define association here
+      User.belongsTo( models.Role, { as: 'role' } );
     }
   };
   User.init({
     firstName: DataTypes.STRING,
-    lastName: DataTypes.STRING,
+    LastName: DataTypes.STRING,
```
Comment (maxxCeballos) [inline models/user.js]: usar minusculas como primera letra de atributo

**Review by maxxCeballos — CHANGES_REQUESTED** [review]

**Comment by sebasam** [comment]
### Summary

Capital letter from LastName has been removed for lastName, sorry for that issue

**Review by maxxCeballos — APPROVED** [review]

**Pull request merged**